Switched to branch 'daily'
Client-Request-ID=1bf28c3c-6cfa-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Sun, 10 Jun 2018 22:03:29 GMT, Server-Request-ID=7ece0cde-401e-0020-3006-015e0e000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:7ece0cde-401e-0020-3006-015e0e000000Time:2018-06-10T22:03:30.2440467Z</Message></Error>.
Client-Request-ID=1bf28c3e-6cfa-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Sun, 10 Jun 2018 22:03:29 GMT, Server-Request-ID=8cc6e38b-301e-0005-0206-01f772000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:8cc6e38b-301e-0005-0206-01f772000000Time:2018-06-10T22:03:30.4043762Z</Message></Error>.
Client-Request-ID=1bf28c40-6cfa-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Sun, 10 Jun 2018 22:03:30 GMT, Server-Request-ID=d58cfdf0-c01e-0063-3b06-01b852000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:d58cfdf0-c01e-0063-3b06-01b852000000Time:2018-06-10T22:03:30.5673669Z</Message></Error>.
Client-Request-ID=1bf28c42-6cfa-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Sun, 10 Jun 2018 22:03:29 GMT, Server-Request-ID=be7ab352-e01e-0074-3c06-011159000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:be7ab352-e01e-0074-3c06-011159000000Time:2018-06-10T22:03:30.7266386Z</Message></Error>.
Traceback (most recent call last):
  File "./02.local.py", line 92, in <module>
    min(metrics['mse']), 
KeyError: 'mse'
Client-Request-ID=8e52fa88-6cfe-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Sun, 10 Jun 2018 22:35:19 GMT, Server-Request-ID=ea1cecae-e01e-0029-390b-011bdd000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:ea1cecae-e01e-0029-390b-011bdd000000Time:2018-06-10T22:35:20.1232973Z</Message></Error>.
Client-Request-ID=8e52fa8a-6cfe-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Sun, 10 Jun 2018 22:35:20 GMT, Server-Request-ID=c44efd54-501e-002c-590b-01c906000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:c44efd54-501e-002c-590b-01c906000000Time:2018-06-10T22:35:20.2833888Z</Message></Error>.
Client-Request-ID=8e52fa8c-6cfe-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Sun, 10 Jun 2018 22:35:20 GMT, Server-Request-ID=31e9bc67-801e-0062-3b0b-01e78e000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:31e9bc67-801e-0062-3b0b-01e78e000000Time:2018-06-10T22:35:20.4386702Z</Message></Error>.
Client-Request-ID=8e52fa8e-6cfe-11e8-9288-000d3a71b7c3 Retry policy did not allow for a retry: Server-Timestamp=Sun, 10 Jun 2018 22:35:20 GMT, Server-Request-ID=6a8928ec-801e-0072-4b0b-0122e6000000, HTTP status code=409, Exception=The specified container already exists.ErrorCode: ContainerAlreadyExists<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerAlreadyExists</Code><Message>The specified container already exists.RequestId:6a8928ec-801e-0072-4b0b-0122e6000000Time:2018-06-10T22:35:20.5984963Z</Message></Error>.
Traceback (most recent call last):
  File "./10.o16n.py", line 88, in <module>
    model.delete()
  File "/home/ninghai/miniconda3/envs/daily/lib/python3.6/site-packages/azureml/core/model.py", line 313, in delete
    'Content: {}'.format(resp.status_code, resp.headers, resp.content))
azureml.exceptions.azureml_exception.WebserviceException: Received bad response from Model Management Service:
Response Code: 412
Headers: {'Date': 'Sun, 10 Jun 2018 22:40:42 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'api-supported-versions': '2018-03-01-preview', 'x-ms-client-request-id': '86d86cdab79146c38c0d0322b190beab', 'x-ms-client-session-id': '', 'Strict-Transport-Security': 'max-age=15724800; includeSubDomains; preload'}
Content: b'{"code":"PreconditionFailed","statusCode":412,"message":"PreconditionFailed","details":[{"code":"DeletionRequired","message":"The model with id mymodel:1 can not be deleted because it is currently being used in one or more images"}]}'
Switched to branch 'master'
From github.com:Azure/hai
   b27e2a7..0e20284  master     -> origin/master
